server/services/status-event-tracker.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var getData    = require('../business-process-query/get-data')
+  , getDateMap = require('../business-process-query/get-status-history-date-map');
```

Comment:
We should not create a service for that, but initialize it in base utility that needs it.

e.g. `getData` is initialized in statistics routes  -> https://github.com/egovernment/eregistrations/blob/master/server/routes/statistics.js#L66

The `getDateMap` should be initialized most likely in body of `calculate-status-events-sums.js`